_fossfund/15-barbel.md

```diff
@@ -0,0 +1,10 @@
+---
+title: BARBEL
+logo: balbel.png
+projectUrl: https://babeljs.io
```

Comment:
This [image ](https://raw.githubusercontent.com/babel/logo/master/babel.png)is broken, this image should be added to https://github.com/microsoft/opensource.microsoft.com/tree/1dbc9603ccbda0f41c0535267ca502abe80aa0aa/assets/images/fossfund